xml/deploy_cephadm.xml

```diff
@@ -179,6 +191,24 @@
      Install the package <literal>salt-minion</literal> on all minion nodes.
     </para>
 <screen>&prompt.sminion;zypper in salt-minion</screen>
+   </step>
+   <step>
+     <para>
+       Edit <filename>/etc/salt/minion</filename> and uncomment the following
+       line:
+     </para>
+<screen>#log_level: WARN</screen>
```

Comment:
I'm thinking we should change `log_level` to `log_level_logfile` here

Reply:
I think we only want to change `log_level_logfile`... but need to test it.